### PR TITLE
 change email, can not use asml company email

### PR DESCRIPTION
**Pull request by naijgnay**

### CODE REVIEW

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java [hunk 1]
```diff
@@ -35,6 +41,22 @@
     return webAgentSessionRegistry.getRooms();
   }
 
+  @ResponseBody
+  @PostMapping("/api/rooms/{roomName}")
+  public ResponseResult checkRoomPassword(@RequestBody Room room, HttpServletResponse response) {
+    Optional<Room> findRoom = webAgentSessionRegistry.getRooms().stream().filter(roomItem -> (room.getName().equals(roomItem.getName()) && (room.getRoomPassword().equals(roomItem.getRoomPassword())))).findAny();
+      if(findRoom.isPresent()) {
+        String cookieKey = "roomPassword_".concat(room.getName());
```
Comment (TRUEJASONFANS) [inline sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java]: Please concat the room Id as cookie key

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java [hunk 2]
```diff
@@ -35,6 +41,22 @@
...
+        String cookieValue = room.getRoomPassword();
+        Cookie passwordCookie = new Cookie(cookieKey, cookieValue);
+        passwordCookie.setPath("/");
```
Comment (TRUEJASONFANS) [inline sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java]: why not set more specific?

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java [hunk 3]
```diff
@@ -35,6 +41,22 @@
...
   }
 
+  @ResponseBody
+  @PostMapping("/api/rooms/{roomName}")
+  public ResponseResult checkRoomPassword(@RequestBody Room room, HttpServletResponse response) {
+    Optional<Room> findRoom = webAgentSessionRegistry.getRooms().stream().filter(roomItem -> (room.getName().equals(roomItem.getName()) && (room.getRoomPassword().equals(roomItem.getRoomPassword())))).findAny();
+      if(findRoom.isPresent()) {
+        String cookieKey = "roomPassword_".concat(room.getName());
+        String cookieValue = room.getRoomPassword();
+        Cookie passwordCookie = new Cookie(cookieKey, cookieValue);
```
Comment (TRUEJASONFANS) [inline sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java]: How about invalida time for the cookie?

Comment (naijgnay) [reply sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java]: 默认和session的过期时间一致，当浏览器关闭的时候失效

**Pull request merged**